pages/project.js

```diff
@@ -0,0 +1,51 @@
+import styles from '../components/landing_page.module.css'
+import p from '../components/project_page.module.css'
+
+function Project() {
+  return (
+    <div className={p.body} id={p.bg}>
+        <div className={p.head_rect}>
+          <h1 className={p.txt} id={p.title}>Firehouse Spice Company</h1>
+          <a href="https://firehousespiceco.com" className={p.txt} id={p.sub}>firehousespiceco.com</a>
+        </div>
+        <div className={p.row}>
```

Comment:
![Screen Shot 2020-08-03 at 12 02 43 PM](https://user-images.githubusercontent.com/50561164/89202831-89588a00-d581-11ea-930d-f09ef4d1da63.png)
Rows get a bit funky when resized to mobile. The bottom image seems to stretch to the width of the screen while the "row" div is much smaller.